extensions/game.py

```diff
@@ -103,52 +103,15 @@ def godfather(self) -> _players.Player:
     # Notification methods
 
     async def night_notification(self):
-        embed = discord.Embed(
-            title=f"Night {self._day - 1}",
-            description="Check your private channels",
-            colour=0x0A0A86,
-        )
-        embed.set_thumbnail(
-            url="https://www.jing.fm/clipimg/full/132-1327252_half-moon-png-images-moon-clipart-png.png"
-        )
-        await self.chat.send(content=self._alive_game_role.mention, embed=embed)
-
-    async def day_notification(self, *notifications: str):
-        """Creates a notification embed with all of todays notifications"""
-        msg, current_notifications = self._day_notifications.get(self._day, (None, []))
-        current_notifications.extend(notifications)
-        fmt = "Roles Alive:\n"
-        # Get alive players to add to alive roles
-        alive_players = {}
-        for player in self.players:
-            if player.dead:
-                continue
-            alive_players[str(player)] = alive_players.get(str(player), 0) + 1
-        fmt += "\n".join(f"{key}: {count}" for key, count in alive_players.items())
-        fmt += "\n\n"
-        # If we're not on day one, notify that you can nominate
-        if self._day > 1:
-            fmt += f"**Type >>nominate member to nominate someone to be lynched**. Chat in {self.chat.mention}\n\n"
-        else:
-            fmt += f"Chat in {self.chat.mention}\n\n"
-        # Add the recent actions
-        fmt += "**Recent Actions**\n"
-        fmt += "\n".join(current_notifications)
+        async with self.chat.typing():
+            buffer = await self.ctx.bot.create_night_image(self)
+            await self.chat.send(file=discord.File(buffer, filename="night.png"))
 
-        embed = discord.Embed(
-            title=f"Day {self._day}", description=fmt, colour=0xF6F823
-        )
-        embed.set_thumbnail(
-            url="https://media.discordapp.net/attachments/840698427755069475/841841923936485416/Sw5vSWOjshUo40xEj-hWqfiRu8Ma2CtYjjh7prRsF6ADPk_z7znpEBf-E3i44U9Hukh3ZJOFhm9S43naa4dEA8pXX4dfAJeEv0bl.png"
-        )
-        if msg is None:
-            msg = await self.info.send(
-                content=self._alive_game_role.mention, embed=embed
-            )
-        else:
-            await msg.edit(embed=embed)
-
-        self._day_notifications[self._day] = [msg, current_notifications]
+    async def day_notification(self, *deaths: Player):
+        """Creates a notification imdage with all of todays notifications"""
```

Comment:
image* and since it's not just a list of notifications like it used to, it probably fits to say something more like "Creates a notification image that show the deaths that happened the night prior"